sdk/ml/azure-ai-ml/tests/dsl/unittests/test_mldesigner_imports.py

```diff
@@ -34,16 +34,24 @@ def test_necessay_attributes(self):
         assert hasattr(PipelineComponent, "_to_dict")
         assert hasattr(PipelineComponent, "_source_path")
         assert hasattr(PipelineComponent, "jobs")
-        assert hasattr(InternalComponent, "_to_dict")
-        assert hasattr(InternalComponent, "_source_path")
-        assert hasattr(InternalComponent, "_additional_includes")
-        assert hasattr(InternalAdditionalIncludes, "with_includes")
-        assert hasattr(InternalAdditionalIncludes, "code_path")
-        assert hasattr(InternalAdditionalIncludes, "includes")
         assert hasattr(ValidationResult, "passed")
         assert hasattr(ValidationResult, "error_messages")
         assert hasattr(ParallelFor, "_to_rest_items")
 
+    @pytest.mark.usefixtures("enable_internal_components")
```

Comment:
Should we also test function `_get_all_additional_includes_configs` since that's also called in mldesigner code?

Reply:
Can refert to the change in this PR: https://msdata.visualstudio.com/DefaultCollection/Vienna/_git/sdk-cli-v2/pullrequest/1080655?path=/src/mldesigner/mldesigner/_compile/_base_compiler.py&_a=files

![image](https://github.com/Azure/azure-sdk-for-python/assets/2572521/cfe75563-6b9f-4e8e-bd8c-b77b9f2c10ca)

Now we should guard the function and make sure the returned object is a list with additional includes file names.

Reply:
Given old interface are protected, I think we can revert changes on mldesigner